### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

**Review by Ahets — CHANGES_REQUESTED** [review]
Add empty row

**Review by bbondarets — CHANGES_REQUESTED** [review]
add new empty line in the end of file

**Review by YuraVasko — CHANGES_REQUESTED** [review]
Add new line at the end of file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (kravchenkov) [inline index.html]: You need add to new line.

**Review by pyaryna — CHANGES_REQUESTED** [review]
Please add new line at the end of the file.

**Review by kravchenkov — CHANGES_REQUESTED** [review]
You need add to new line.

**Review by solayusko — CHANGES_REQUESTED** [review]
Please, add a new line on the end

**Review by NazarPolevyi — COMMENTED** [review]
You need to add new line in the end.

**Review by yurakozlovskiy — COMMENTED** [review]
add empty row

**Review by solayusko — APPROVED** [review]

**Review by YuraVasko — APPROVED** [review]

**Review by pyaryna — APPROVED** [review]

**Review by bbondarets — APPROVED** [review]
Nice work

**Review by Ahets — APPROVED** [review]

**Review by kravchenkov — APPROVED** [review]

**Review by SofiiaFlys — APPROVED** [review]

**Pull request merged**